aptos-move/framework/aptos-stdlib/sources/table_with_length.spec.move

```diff
@@ -4,23 +4,11 @@ spec aptos_std::table_with_length {
     // Make most of the public API intrinsic. Those functions have custom specifications in the prover.
 
     spec TableWithLength {
-        pragma intrinsic = map,
-            map_new = new,
-            map_destroy_empty = destroy_empty,
-            map_len = length,
-            map_is_empty = empty,
-            map_has_key = contains,
-            map_add_no_override = add,
-            map_add_override_if_exists = upsert,
-            map_del_must_exist = remove,
-            map_borrow = borrow,
-            map_borrow_mut = borrow_mut,
-            map_borrow_mut_with_default = borrow_mut_with_default,
-            map_spec_get = spec_get,
-            map_spec_set = spec_set,
-            map_spec_del = spec_remove,
-            map_spec_len = spec_len,
-            map_spec_has_key = spec_contains;
+        pragma intrinsic = map, map_new = new, map_destroy_empty = destroy_empty, map_len =
```

Comment:
We should preserve the original line break here.

Reply:
we fixed at: https://github.com/movebit/movefmt/commit/2bf7f82ca12800f4af8796fe8d3c53f303a5695e